index.html

```diff
@@ -122,6 +125,37 @@ <h2>A project for first-time contributions</h2>
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+
+   <!-- ________ Joseph adeniji  card START ________  -->
+
+        <div class="card">
+          <p class="name">Joseph adeniji</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://twitter.com/joshiz02" target="_blank">Your handle</a>
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/jophizz" target="_blank">Your handle</a>
```

Comment:
Enter your user handle in the `Your handle` space as well since that displays that the link redirects to your user handles.